app/routes/application.js

```diff
@@ -93,14 +93,14 @@ export default Route.extend(
 			) {
 
 				window.waitForAds((adEngine3Loaded) => {
+					const adsModule = this.get('ads.module');
+
+					adsModule.init(model.adsContext);
```

Comment:
There is no adsContext at this point. AE3 has to be initialised in wiki-page.js

https://github.com/Wikia/mobile-wiki/pull/621/files#diff-d77b3e7a178bee9c8affd145b2261a54R131